addons/analytic/views/analytic_account_views.xml

```diff
@@ -282,7 +282,7 @@
                     <group>
                         <group name="analytic_entry" string="Analytic Entry">
                             <field name="name"/>
-                            <field name="account_id"/>
+                            <field name="account_id" options="{'no_create': True}"/>
```

Comment:
this is an analytic account, if I'm not wrong... so we probably don't want to restrict its creation here